src/components/Footer.messages.js

```diff
@@ -103,7 +103,7 @@ const messages = defineMessages({
   },
   'footer.legalLinks.doNotSellData': {
```

Comment:
Given the default message change, wouldn't the `id`, `footer.legalLinks.doNotSellData`, and `CCPA_COOKIE_NAME` want to reflect that?

Reply:
I believe this is still related to DNS and CCPA, since basically setting this cookie will trigger the LDU flag from Facebook. I did update the description to reflect better on what the link does now.